ansible/wipe.yml

```diff
@@ -29,7 +29,6 @@
     retries: 10
     delay: 3
     until: delete_task is success
-
   - name: "wipe grafana db"
```

Comment:
It seems `wipe` currently does not stop services / erase the files associated with the ELK stack. If you believe it is non-trivial work to incorporate this now, we should at least log an issue to track this as a known gap and try to address it in a future PR.

Reply:
I will create an issue for this.

Reply:
Resolving as #368 tracks this.